src/main/webapp/category.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>category</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="fetch_category.js"></script>
+  </head>
+  <body style="color: aliceblue;">
+    <nav class="nav">
```

Comment:
This is a nifty way to only write the nav bar HTML once and re-use it across all the pages: https://stackoverflow.com/questions/31954089/how-can-i-reuse-a-navigation-bar-on-multiple-pages

Not going to block on this change, but it would be super useful for iterating on the nav bar.

Reply:
Oh wait, we are not using JQuery, so maybe the second comment here is the easiest way to do it with `document.getElementById('nav').innerHTML = html;` line in the JS file.